src/geometry/util.rs

```diff
@@ -40,12 +39,12 @@ pub struct Distributor {
 }
 
 impl Distributor {
-    pub fn distribution(&self) -> HashMap<u32, Body> {
-        let mut result: HashMap<u32, Body> = HashMap::new();
+    pub fn distribution(&self) -> Vec<Body> {
+        let mut result: Vec<Body> = vec![];
         let mut angle_rand = thread_rng();
         let mut dist_rand = thread_rng();
 
-        for i in 0..self.num_bodies {
+        for _ in 0..self.num_bodies {
```

Comment:
`_` _ instead of i ? i havent seen that before ^^

Reply:
We no longer need to use the value `i`, so we can just write `_` to mean "we don't care about the value". You do this when you just need to go through a loop a certain number of times, but don't care about the index.